client/src/templates/Challenges/classic/editor.css

```
@@ -269,6 +269,11 @@ textarea.inputarea {
   cursor: pointer;
 }
 
+.code-details-overflow-container {
+  max-width: 100%;
+  overflow-x: auto;
+}
+
```

```suggestion
.editor-upper-jaw pre code {
  white-space: pre;
}

```
This css change should be enough.

---

The rest of the changes can be reverted.